spec/CampMinder/EstablishConnection_spec.rb

```diff
@@ -3,19 +3,18 @@
 describe CampMinder::EstablishConnection do
   before do
     @data = {
-      'clientID' => 'C-123',
-      'personID' => 'P-123',
+      'client_id' => 'C-123',
+      'person_id' => 'P-123',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.